src/Altinn.App.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -322,7 +321,7 @@ private static void AddExternalApis(IServiceCollection services)
 
     private static void AddProcessServices(IServiceCollection services)
     {
-        services.TryAddTransient<IProcessEngine, ProcessEngine>();
+        services.TryAddScoped<IProcessEngine, ProcessEngine>();
```

Comment:
Seems like this will break 1 app - digdir/oed

[DeceasedStateMiddleware](https://altinn.studio/repos/digdir/oed/src/commit/67bd185a198cbf380d72c173ae4fd73edc8ea953/App/Middlewares/DeceasedStateMiddleware.cs#L25) -> [StorageService](https://altinn.studio/repos/digdir/oed/src/commit/67bd185a198cbf380d72c173ae4fd73edc8ea953/App/Services/StorageService.cs#L38)

Since that type of middleware is a singleton, and they don't inject from the HttpContext. Should we just remember to notify them when we release? I don't think any of the other DI changes here affect anyone

Reply:
I reverted the `AddScoped` changes. They are no longer needed after I rejected the idea of caching data element downloads using a scoped service.

I opened https://github.com/Altinn/app-lib-dotnet/issues/751, because I think we should default more to using scoped services.